analysis/src/topObjectSelection.cc

```diff
@@ -12,11 +12,13 @@ vector<TParticle> topObjectSelection::elecSelection() {
   vector<TParticle> elecs; 
   for (UInt_t i = 0; i < nElectron; ++i){
     if (Electron_pt[i] < 20) continue;
-    if (isSemiLep) { if (Electron_pt[i] < 30) continue; }
-    if (std::abs(Electron_eta[i]) > 2.4) continue;
-    if (Electron_cutBased[i] < 3) continue; 
+    if (isSemiLep) { if (Electron_pt[i] <= 34) continue; }
+    if (std::abs(Electron_eta[i]) > 2.1) continue;
+    if (Electron_cutBased[i] < 4) continue; 
     float el_scEta = Electron_deltaEtaSC[i] + Electron_eta[i];
     if ( std::abs(el_scEta) > 1.4442 &&  std::abs(el_scEta) < 1.566 ) continue;
+    //if ( std::abs(el_scEta) >= 1.566 ) continue; // For AN-2017/083; it must be turned off when QCD-studying
+    if ( Electron_pfRelIso03_all[ i ] > 0.0588 ) continue;
```

Comment:
Same comments as for muon case, we need to think also about the dilepton selections